### PR TITLE
Cloudcashbox documentation

### PR DESCRIPTION
**Pull request by Bluette-C-Riviere**
CloudCashbox documentation

### CODE REVIEW

File: technical-operations/middleware/launchers/Cloudcashbox.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+:::info summary
+
+After reading this, you can explain what the _CloudCashbox_ solution is, how it is used to host the Middleware.
+
+:::
+
+## Introduction
+The CloudCashbox is a fully hosted SaaS version of fiskaltrust's Middleware, offering a centralized and reliable fiscalization solution. Hosted on Microsoft Azure in Germany, it simplifies the complexity of fiscal compliance by eliminating the need for local software configuration and installation. Users can easily connect to the Middleware using an access token and URL, without the need to manage TSEs.
+
+###Key features include:
```
Comment (FlorianStadlberger) [inline technical-operations/middleware/launchers/Cloudcashbox.md:17]: missing space between # and Key

Comment (Bluette-C-Riviere) [reply technical-operations/middleware/launchers/Cloudcashbox.md]: Thanks for the review :) 
Change made with commit b1008d2

File: _markets/de/buy-resell/products/_hosted-middleware.mdx [hunk 2]
```diff
@@ -1 +1,5 @@
-*Due to national regulations, fiskaltrust does not yet offer a hosted Middleware solution in Germany.*
\ No newline at end of file
+### CloudCashbox
+
+*CloudCashbox* is fiskaltrust's SaaS Middleware platform.
+
+It offers the same features as a locally hosted Middleware instance but without needing to install or maintain any software or TSE on your side. Your POS System requires a working Internet connection, which can communicate with *SignatureCloud* over an encrypted, **HTTPS-secured** connection.
```
Comment (volllly) [inline _markets/de/buy-resell/products/_hosted-middleware.mdx:5]:
```suggestion
It offers the same features as a locally hosted Middleware instance but without needing to install or maintain any software or TSE on your side. Your POS System requires a working Internet connection, which can communicate with the *CloudCashbox* over an encrypted, **HTTPS-secured** connection.
```

**Review by volllly — APPROVED** [review]

**Comment by TSchmiedlechner** [comment]
Hey @Bluette-C-Riviere, two general thoughts:
1. Please use lower-case filenames with dashes
2. You'll need to add the newly added files to [toc.js](https://github.com/fiskaltrust/rollout-doc/blob/main/toc.js)

**Review by saschaknapp — APPROVED** [review]

**Review by saschaknapp — APPROVED** [review]

**Pull request merged**